### PR TITLE
99 email validation

### LINKED ISSUE #99: User email validation
Ensure that the user email is always required, perform validation in user views

### PR DESCRIPTION
**Pull request by Nkoli**
Fixes #99

### CODE REVIEW

File: app/serializers.py [hunk 1]
```diff
@@ -81,7 +85,7 @@ def update(self, instance, validated_data):
             instance.spoken_languages.add(*spoken_languages)
 
         instance.username = validated_data.get("username", instance.username)
-        instance.email = validated_data.get("email", instance.email)
+        # instance.email = validated_data.get("email", instance.email)
```
Comment (kosyfrances) [inline app/serializers.py]: You need add email to "instance".

File: app/tests/test_user.py [hunk 2]
```diff
@@ -51,22 +51,22 @@ def test_create_new_user(self):
...
+    # def test_update_fields_in_single_user(self):
+    #     self.client.credentials(HTTP_AUTHORIZATION="Bearer " + self.token)
+    #     response = self.client.put(
+    #         f"/users/{self.profile.user_id}/",
+    #         {
+    #             "role": [self.role2.id],
+    #             "username": "testusertwo",
+    #             "skills": [self.skill2.id],
+    #             "timezone": "Africa/Cairo",
+    #         },
```
Comment (kosyfrances) [inline app/tests/test_user.py]: When doing a put or edit, you also need to ensure that required fields like email are sent along

**Review by kosyfrances — CHANGES_REQUESTED** [review]

**Review by kosyfrances — COMMENTED** [review]
LGTM but add a test checking for when the email is empty in github oauth

**Review by kosyfrances — APPROVED** [review]
